app/views/ncr/proposals/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <h4>National Capital Region</h4>
   <h3>Purchase Card Approval Request</h3>
 
-  <%= form_for @proposal_form do |f| %>
+  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
```

Comment:
The problem is that a proposal form is never actually saved anywhere, so regardless of if we are "editing" or not, form_for will think we're creating a new instance. If there was some flag on the model itself that I could say "hey, pretend live you're going to be edited", that'd be much cleaner. Does that exist?